app/src/main/java/no/usn/mob3000/ui/screens/chess/PlayScreen.kt

```diff
@@ -107,8 +105,7 @@ fun PlayScreen(
                     )
                 }
             }
-
-            Card(colors = CardDefaults.cardColors(containerColor = DefaultListItemBackground)) {
+            Card( colors = CardDefaults.cardColors(containerColor = MaterialTheme.colorScheme.primary)) {
```

Comment:
Same as [with this comment](https://github.com/frigvid/mob3000-group11/pull/27/files#r1847359334). There's an additional space after the starting parenthesis.